From 14aa584f6d005b787b096ab219055f12a41469bd Mon Sep 17 00:00:00 2001
From: Jason King <jason.king@joyent.com>
Date: Tue, 31 Jul 2018 21:43:16 +0000
Subject: [PATCH] OS-7100 Build fails in libvscan after merge Reviewed by: Dan
 McDonald <danmcd@joyent.com> Approved by: Robert Mustacchi <rm@joyent.com>

---
 usr/src/lib/libvscan/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/lib/libvscan/Makefile b/usr/src/lib/libvscan/Makefile
index 1cf8f5effa..1212cc6a73 100644
--- a/usr/src/lib/libvscan/Makefile
+++ b/usr/src/lib/libvscan/Makefile
@@ -42,7 +42,7 @@ MSGFILES =	common/libvscan.c
 
 .KEEP_STATE:
 
-all clean clobber install lint _msg : $(SUBDIRS)
+all clean clobber install lint: $(SUBDIRS)
 
 $(POFILE):	$(MSGFILES)
 	$(BUILDPO.msgfiles)
-- 
2.21.0

